tests/baselines/reference/booleanLiteralTypes1.types

```diff
@@ -1,30 +1,30 @@
 === tests/cases/conformance/types/literal/booleanLiteralTypes1.ts ===
 type A1 = true | false;
->A1 : boolean
+>A1 : A1

```

Comment:
Is this just a bug in our type writer for tests, or is this what quickinfo would show as well (i.e., `type A1 = A1`)?